pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -709,7 +709,8 @@ def _grab_objects(self):
         self._url_type_combo_box = self.builder.get_object("urlTypeComboBox")
         self._url_type_label = self.builder.get_object("urlTypeLabel")
 
-        self._no_updates_checkbox = self.builder.get_object("noUpdatesCheckbox")
+        self._updates_radio_button = self.builder.get_object("updatesRadioButton")
+        self._no_updates_radio_button = self.builder.get_object("noUpdatesRadioButton")
```

Comment:
It looks like this attribute is not used.

Reply:
Indeed! Fixed.